app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_secure_password
```

Comment:
So i have the relationships set up such that the user can see all of their items through the relationships, but might be easier to connect the user closer to those items in later iterations